### PR TITLE
Decouple CMS and OPL emulations from dosbox-staging

### PR DESCRIPTION
**Pull request by RNMB15**
Add a summary of the change(s) brought by this PR here.

## What issue(s) does this PR address?

If there are any related issues, reference them here. Ex. #1234, Fixes #1234, Closes #1234, etc.

## Does this PR introduce new feature(s)?

If yes, describe the feature(s) introduced.

## Does this PR introduce any breaking change(s)?

If yes, describe the breaking change(s) in detail.

## Additional information

Add any additional information here.

### CODE REVIEW

**Comment by RNMB15** [comment]
Decouple CMS and OPL emulations from dosbox-staging.

https://github.com/japsmits/dosbox-staging/commit/bc7d5fd88b34fad257a97db5e2b97e1892ded59a

File: src/dosbox.cpp [hunk 1]
```diff
@@ -1393,6 +1393,7 @@ void DOSBOX_SetupConfigSections(void) {
     const char* gustypes[] = { "classic", "classic37", "max", "interwave", nullptr };
     const char* sbtypes[] = { "sb1", "sb2", "sbpro1", "sbpro2", "sb16", "sb16vibra", "gb", "ess688", "ess1688", "reveal_sc400", "none", nullptr };
     const char* oplmodes[]={ "auto", "cms", "opl2", "dualopl2", "opl3", "opl3gold", "none", "hardware", "hardwaregb", "esfm", nullptr };
+    const char* Set_CMS[]"on", "off", "auto"});
```
Comment (maron2000) [inline src/dosbox.cpp:1396]: const char* Set_CMS[]={"on", "off", "auto"});

**Comment by Allofich** [comment]
You put things within switch statements. When I wrote things like "immediately after the switch," I didn't mean immediately after the line with "switch" in it, I meant after the closing brace of the switch statement. Also you put `bool cms;` in a switch statement, instead of in `struct SB_INFO`.

Another thing is that descriptive commit messages, rather than the "Update <filename>" ones that you get through the GitHub editor, are better. You can make those with a program like Git Bash, although getting that set up and learning to use it can take some time.

Anyway, I think it would be better to let me make a new PR with the changes. I'll try to do it soon.

**Comment by joncampbell123** [comment]
It looks like CI failed. Is this pull request separate from the other one about CMS and OPL?

**Comment by joncampbell123** [comment]
Should I close this one and merge the newer pull request? At a glance they do basically the same thing, except that this failed the build test.

**Comment by Allofich** [comment]
Yes, you can close this one.

**Pull request closed without merging**